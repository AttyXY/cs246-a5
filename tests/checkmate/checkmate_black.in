setup
+ k a8
+ K h1
+ R b1
+ R b2
= white
done
game human human
move b1 a1
