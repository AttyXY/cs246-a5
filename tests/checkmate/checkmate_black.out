WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  k-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  --------
   abcdefgh

8  k-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  -------K
   abcdefgh

8  k-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  -R-----K
   abcdefgh

8  k-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  -R------
1  -R-----K
   abcdefgh

STARTING GAME!

8  k-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  -R------
1  -R-----K
   abcdefgh

8  k-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  -R------
1  R------K
   abcdefgh

Black is in check.
Checkmate! White wins!
Final Score:
White: 1
Black: 0
THANKS FOR PLAYING CHESS!
